include/ddprof_worker_context.hpp

```diff
@@ -30,4 +31,5 @@ struct DDProfWorkerContext {
       cycle_start_time;  // time at which current export cycle was started
   int64_t send_nanos;    // Last time an export was sent
   uint32_t count_worker; // exports since last cache clear
+  std::array<uint64_t, MAX_TYPE_WATCHER> lost_events_per_watcher;
```

Comment:
Oh, this is a good idea!  In fact, maybe we should generalize this a bit in a future PR, emitting counts for each individual state (total frames, total samples, truncations, incompletes, lost, etc)